.github/workflows/verify_consumer_pacts.yaml

```diff
@@ -0,0 +1,203 @@
+name: Verify consumer pacts
+# The purpose of this workflow is to verify ANY consumer contract(s) dependent on workspacemanager provider using Pact
```

Comment:
Comment adjusted to include `workspacemanager` pacticipant name, which must match the helmchart name.